css/style.css

```diff
@@ -2489,7 +2489,7 @@ footer .grid--4 {
   }
 
   #post-results.grid-view {
```

Comment:
@Right-Noe this rule already exists on line 1644

Reply:
I'm sorry if I'm not understanding you correctly, 1644 desktop version, 2491 mobile version. Do you see any issue?

Reply:
My bad, yet it does affect multiple places.
https://news.mongabaydev.wpengine.com/by/elizabeth-claire-alberts/
https://news.mongabaydev.wpengine.com/?s=&formats=post
While fixing one the other one is broken (search).

Reply:
I'm going to use this, are you okay with that?
 #post-results.grid-view {
    grid-template-columns: 1fr;
    gap: 20px;
    margin-top: var(--spacing-10);
  }

  .tax-byline #post-results.grid-view, 
  .author #post-results.grid-view{
    grid-template-columns: repeat(2, 1fr);
  }

Reply:
I would rather change template itself rather than chaining CSS classes. I updated template file and it should have general classes.

Reply:
Yes, sorry, I've seen your comment below :)